servers/quarkus-server/src/test/java/org/projectnessie/server/authz/TestAuthzMeta.java

```diff
@@ -120,8 +120,7 @@ void beforeEach() {
   @Test
   public void icebergApiTable() {
     var catalog =
-        catalog(
-            Map.of("http.header.Authorization", basicAuthorizationHeader("admin_user", "test123")));
+        catalog(Map.of("header.Authorization", basicAuthorizationHeader("admin_user", "test123")));
```

Comment:
The changes around the `http.` prefix are a bit suspicious.

Maybe @adutra can take a look at it in a follow-up.